.packit.yaml

```diff
@@ -28,12 +28,18 @@ jobs:
   - job: copr_build
     trigger: pull_request
     targets:
-      centos-stream-8:
+      rhel-8:
         additional_modules: "katello:el8,foreman-devel:el8"
         additional_repos:
           - http://yum.theforeman.org/releases/nightly/el8/x86_64/
           - http://yum.theforeman.org/plugins/nightly/el8/x86_64/
           - http://yum.theforeman.org/katello/nightly/katello/el8/x86_64/
+      rhel-9:
+        additional_modules: "katello:el9,foreman-devel:el9"
```

Comment:
On EL9, you don't have to enable DNF modules: https://docs.theforeman.org/nightly/Installing_Server/index-katello.html#repositories-el-9

packit log:

````
Error: Problems in request:
missing groups or modules: katello:el9
WARNING: DNF4 command failed, retrying, attempt #2, sleeping 10s
No matches found for the following disable plugin patterns: local, spacewalk, versionlock
````

Reply:
Packit works if I add the Zeitwerk fix to it... What shall we do? 1) I can remove the Zeitwerk commits and we can merge this PR as is or 2) I can add the commit from this PR to the Zeitwerk PR #141 .